src/main/resources/graphql/offender.graphqls

```diff
@@ -1,6 +1,4 @@
 type Query {
-    # Get a list of offenders
-    offenders: [Offender]
     # Get a list of offenders last name
```

Comment:
didn't make sense to have a query that could return ALL offenders given this now points are a real database